elastic/integrations
# Enhancements dashboards azure

## What does this PR do?

List of changes - Dashboard enhancements in the Azure package implementation: 

1. Inlining the visualizations
2. Set up global filters on the relevant dataset used in the dashboard
3. Replace deprecated input control and reorganize the panels if necessary 
4. Bumping up the version of the search panels 

Azure contains:
- Generic EventHub
- Azure Active Directory Logs ( Sign-in, Audit, Identity Protection, Provisioning)
- Azure Activity Logs 
- Azure Platform Logs
- Azure Spring Cloud logs
- Azure Firewall logs
- Azure Application Gateway Logs

List of enhanced dashboards:
- [Logs Azure] Azure Spring Cloud logs Overview
- [Logs Azure] Azure Spring Cloud Logs System Logs
- [Logs Azure] Alerts Overview
- [Logs Azure] User Activity
- [Logs Azure] Cloud Overview
- [Logs Azure] Firewall DNS Proxy Log
- [Logs Azure] Firewall Application Rule Log
- [Logs Azure] Firewall Network NAT Rule Log
- [Logs Azure] Azure AD Provisioning Logs
- [Logs Azure] Azure AD Identity Protection
- [Logs Azure] Azure Spring Cloud Logs Application Cloud Logs
- [Logs Azure] Firewall Overview

## Checklist


## Author's Checklist



## How to test this PR locally

Check to validate changes:
1. Check folder structure and panels are unbroken
2. Check if pre-filters are applied below the search query bar 
3. Validate that the new input controls have the same functionality as the deprecated ones

## Related issues

## Screenshots

Example dashboard to illustrate changes - [Logs Azure] User Activity

Before:
<img width="751" alt="before" src="https://user-images.githubusercontent.com/122360116/233628700-d65c8cbe-db56-4f5d-8c37-e62e2c3f03f1.png">

After:

<img width="754" alt="after" src="https://user-images.githubusercontent.com/122360116/233630507-c787129b-dae5-4abc-9567-0e720ed1bbb0.png">
